### ISSUE TITLE
How to compute and show losses when running validation (val.py) on a testing dataset?

### ISSUE THREAD

**Issue by fvea** [ultralytics/yolov5#10619]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi! I would like to show the classification and localization loss my model has when I evaluate it on my testing dataset using val.py. However, it seems that val.py only shows the object detection metrics. Is there a workaround to this?

 Basically, I would like to display the losses during training as shown in the snippet below:
![image](https://user-images.githubusercontent.com/75005859/209834204-17bb472c-c8cd-43b2-b833-635b0920fd95.png)

to the output of val.py when running it to a testing dataset as shown below:
![eval1](https://user-images.githubusercontent.com/75005859/209834404-950defd7-922e-42de-98b3-29c7d4b87adb.png)

Any help would be greatly appreciated. Thank you so much!

**Comment by utkutpcgl** [comment]
I would appreciate this feature also, thanks.

**Comment by glenn-jocher** [comment]
@utkutpcgl thank you for your suggestion! We understand the importance of being able to display classification and localization losses when running validation on a testing dataset using val.py. At the moment, val.py only shows the object detection metrics. 

We appreciate your feedback and will take it into consideration for future updates. If you have any further suggestions or questions, please don't hesitate to let us know. Your contributions to YOLOv5 are greatly appreciated!

**Comment by utkutpcgl** [comment]
Thanks! Your contribution is invaluable to the computer vision society. I won't hold back if I notice something.

**Comment by glenn-jocher** [comment]
@utkutpcgl thank you for your kind words and support! We greatly appreciate your contributions to the YOLOv5 project and your commitment to improving computer vision in the community. We look forward to any feedback you might have in the future. If you have any further questions or suggestions, please don't hesitate to reach out.
